plugins/woocommerce-blocks/tests/e2e/tests/product-collection/product-collection.block_theme.spec.ts

```diff
@@ -1164,7 +1164,7 @@ test.describe( 'Product Collection', () => {
 	} );
 
 	test.describe( 'Product Collection should be visible after Refresh', () => {
-		test( 'In a Product Archive (Product Catalog)', async ( {
+		test( 'Product Collection should be visible after Refresh in a Template', async ( {
```

Comment:
@imanish003 @gigitux I think adding comments to regression tests would be good practice so they're easily referenced and stay tied to the bug fix. [Here's an example of such a comment](https://github.com/WordPress/gutenberg/blob/e131e6e14cc958a938cd74a277f146a9a98aba06/test/e2e/specs/editor/plugins/inner-blocks-template-sync.spec.js#L103-L104).

What do you think?

https://github.com/WordPress/gutenberg/blob/e131e6e14cc958a938cd74a277f146a9a98aba06/test/e2e/specs/editor/plugins/inner-blocks-template-sync.spec.js#L103-L104

Reply:
That makes sense to me. As this PR is already merged, I will create a new PR with a comment.